syntax = "proto2";

package y2019;

message VisionControl {
  optional bool high_video = 1;
}

message VisionStatus {
  optional uint32 high_frame_count = 1;
  optional uint32 low_frame_count = 2;
}
